src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.cpp

```diff
@@ -1,2214 +1,2222 @@
-        // an exception.
-        if (m_ownContext)
-        {
-            s_scopeContextManager.Get().PopContext();
-        }
-    }
-
-    UiaOperationScope& UiaOperationScope::operator=(UiaOperationScope&& other)
-    {
-        m_ownContext = other.m_ownContext;
-        other.m_ownContext = false;
-        return *this;
-    }
-
-    /* static */ std::shared_ptr<UiaOperationDelegator> UiaOperationScope::GetCurrentDelegator()
-    {
-        return s_scopeContextManager.Get().GetCurrentDelegator();
-    }
-
-    void UiaOperationScope::Resolve()
-    {
-        HRESULT operationResult = S_OK;
-        ResolveHrInternal(operationResult);
-        THROW_IF_FAILED(operationResult);
-    }
-
-    [[nodiscard]] HRESULT UiaOperationScope::ResolveHr() noexcept try
-    {
-        HRESULT operationResult = S_OK;
-        ResolveHrInternal(operationResult);
-        return operationResult;
-    }
-    CATCH_RETURN();
-  
-    void UiaOperationScope::ResolveHrInternal(HRESULT& operationResult)
-    {
-        operationResult = S_OK;
-
-        // Resolve does nothing if we don't own the current context. 
-        if (m_ownContext)
-        {
-            auto delegator = GetCurrentDelegator();
-            if (delegator->GetUseRemoteApi())
-            {
-                for (auto& binding : s_scopeContextManager.Get().GetCurrentBindings())
-                {
-                    binding(this);
-                }
-
-                auto result = delegator->Execute();
-
-                operationResult = result.OperationStatus();
-
-                if (SUCCEEDED(operationResult))
-                {
-                    for (auto& resolver : remoteOperationResolvers)
-                    {
-                        resolver(result);
-                    }
-                }
-            }
-
-            s_scopeContextManager.Get().PopContext();
-            m_ownContext = false;
-        }
-    }   
-
-    UiaOperationScope UiaOperationScope::StartNew()
-    {
-        s_scopeContextManager.Get().PushContext();
-        return UiaOperationScope(true /* ownContext */);
-    }
-
-    UiaOperationScope UiaOperationScope::StartOrContinue()
-    {
-        if (GetCurrentDelegator())
-        {
-            return UiaOperationScope(false /* ownContext */);
-        }
-        return StartNew();
-    }
-
-    UiaOperationScope UiaOperationScope::ContinueIfRemote()
-    {
-        if (ShouldUseRemoteApi())
-        {
-            return UiaOperationScope(false /* ownContext */);
-        }
-        // if there is no active remote operation, create a context that never executes remotely.
-        s_scopeContextManager.Get().PushLocalContext();
-        return UiaOperationScope(true /* ownContext */);
-    }
-
-    bool ShouldUseRemoteApi()
-    {
-        auto delegator = UiaOperationScope::GetCurrentDelegator();
-        return delegator && delegator->GetUseRemoteApi();
-    }
-} // namespace UiaOperationAbstraction
+// Copyright (c) Microsoft Corporation.
```

Comment:
looks like you ended up touching every single line in this file -- some whitespace diff, but a diff nonetheless

any chance you could fix it so that the diff is scoped to just what we're adding here?

Reply:
Hmm... I will take a look.

Reply:
Done.